recipes/semver200/all/test_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake, tools, RunEnvironment
+from conan.tools.build import cross_building
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package_multi"
```
It will generate xxx-config.cmake files.